docs/sources/developers/plugins/_index.md

```diff
@@ -6,77 +6,51 @@ title: Build a plugin
-```
+These are among the types of plugins you can create:
 
-Follow the questions and you will have a starter plugin ready to develop.
+- **Panel plugins** - Visualize data and navigating between dashboards.
+- **Data source plugins** - Link to new databases or other sources of data.
+- **App plugins** - Bundle panels and data sources together to create a better user experience.
+- **Other plugins** - Visualize data source logs or streaming data sources.
 
-If you want a more guided introduction to plugin development, check out our tutorials:
+The possibilities are as endless as your imagination, and you can use this guide to help make your dreams become reality. 
 
-- [Build a panel plugin](/tutorials/build-a-panel-plugin/)
-- [Build a data source plugin](/tutorials/build-a-data-source-plugin/)
+   Note: To learn more about the types of plugins you can build, refer to the [Plugin Overview]({{< relref "../../administration/plugin-management/" >}}) documentation.
 
-## Go further
-
-Learn more about specific areas of plugin development.
-
-### Tutorials
-
-If you're looking to build your first plugin, check out these introductory tutorials:
-
-- [Build a panel plugin](/tutorials/build-a-panel-plugin/)
-- [Build a data source plugin](/tutorials/build-a-data-source-plugin/)
-- [Build a data source backend plugin](/tutorials/build-a-data-source-backend-plugin/)
-
-Ready to learn more? Check out our other tutorials:
+## Contents of this developer's guide
 
-- [Build a panel plugin with D3.js](/tutorials/build-a-panel-plugin-with-d3/)
+The following topics give you an idea of what to expect from this guide:
 
-### Guides
+- [**Get started with plugins** - The easiest way to start developing Grafana plugins is to use the Create-plugin tool. Walk step by step for downloading the tool and using it to get your plugin up and running as fast as possible. 
+- [**Introduction to plugin development** - Ground yourself by learning the key concepts involved in Grafana plugin development. For example, learn about frontend and backend development processe, data frames, and error handling. 
+- [**Create a Grafana plugin** - If you are already familiar with plugin creation basics, then it's time to dive deeper into specific development scenarios. You will find tutorials for creating panel plugins, data source plugins, and more. 
+- [**Migrate a plugin** - Learn how to upgrade from a previous version of a Grafana plugin, rewrite an old Angular plugin in React, or update to a newer version. 
+- [**Publish a Grafana plugin**  - Here you'll find what you need to know if you want to publish a plugin to the Grafana Catalog, including publishing criteria, signing process, and publishing.
+- [**Advanced topics** - Learn all about advanced plugin features such as nested plugins. If you find yourself needing to work with Legacy plugins, check here for recommendations.
+- [**Reference** - Description of the Plugin.json schema and plugin metadata.
```

Comment:
Are the bolded words in these bullets supposed to be links? The previous deleted bullets had links. Are the opening [ left-overs?

Reply:
I will soon check in an update that will make all of these links. I had to build the files and add links to those files.